.flake8

```diff
@@ -0,0 +1,3 @@
+[flake8]
+ignore = E402
+max-line-length=160
```

Comment:
160? Seems a bit high

Reply:
We use 120 across most of the other repos

Reply:
Copied 160 from sdx collect . No probs changing , but with modern screens that fits ok.

Reply:
leaving at 160 until there is agreement cross teams , we have 120 in RAS , 150 in EQ and 160 in SDX.